data/initrd/scripts/url/hmc/menu

```diff
@@ -0,0 +1 @@
+HMC
```

Comment:
What's the reason for saying it's "too long?"  Would
HMC (Hardware Mgmt. Cons)
or
HMC (Hdw. Mgmt. Console)
be better? I tend to agree that it would be better to give _some_ clue here.